src/newgidmap.c

```diff
@@ -46,16 +46,83 @@
  */
 const char *Prog;
 
+enum quadopt_t {
```

Comment:
how is this "quad"?

Reply:
I'm using the term `quadoption` from Mutt, but you're right that this might not be the best name.

Reply:
Actually this looks `triopt`, as it has three values.

In github.com/moby/buildkit we call this `BoolOrAuto`.